js/app/components/chat/chat.tsx

```diff
@@ -9,17 +8,110 @@ import {
   useChat,
   usePlayerLivestream,
 } from "features/player/playerSlice";
-import usePlatform from "hooks/usePlatform";
 import { useEffect, useRef, useState } from "react";
 import { TouchableOpacity, Linking } from "react-native";
 import { useAppDispatch, useAppSelector } from "store/hooks";
-import { Button, ScrollView, Sheet, Text, useMedia, View } from "tamagui";
+
+import {
+  Button,
+  isWeb,
+  ScrollView,
+  Sheet,
+  Text,
+  useMedia,
+  View,
+} from "tamagui";
 import { RichText } from "@atproto/api";
+import { chatEvents } from "./chat-events";
 import { ReactElement } from "react";
+import Storage from "storage";
+
+const REPLY_INFO_STORAGE_KEY = "streamplace_reply_info";
+
+async function storeReplyInfoAsync(
```

Comment:
We can lose this stuff now 'cause replies are on the relay

Reply:
And we can just keep "local-first" message rendering in memory in the Redux state, I don't care if it briefly goes away if you refresh